commune_app/tests/test_user_functions.py

```diff
@@ -61,18 +65,44 @@ def chore0(user0):
     return chore0
 
 
+@pytest.fixture
+@pytest.mark.django_db()
+def chore1(user0):
+    chore1 = Chore(
+        title=TITLE,
+        description=DESCRIPTION,
+        date=DATE,
+        budget=BUDGET,
+        assign_to=user0,
+        passed=NOT_PASSED,
+        completed=COMPLETED
+        )
+    chore1.save()
+    return chore1
+
+
 @pytest.mark.django_db()
 class TestUserFunctions:
 
     def test_join_commune(self, user0, commune0):
-        user0.join_commune(commune0.id)
+        assert user0 not in User.objects.filter(commune_id=commune0)
+        join_string = user0.join_commune(commune0.id)
```

Comment:
usually in this kind of method, we return only true/false
then you can check if the return value is true